addon/actions.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash'
+import {validateValue} from './validator/index'
+import {aggregateResults} from './validator/utils'
+import {getPath} from './dereference'
+
+export const CHANGE_VALUE = 'CHANGE_VALUE'
+export const VALIDATION_RESOLVED = 'VALIDATION_RESOLVED'
+
+export function changeValue (bunsenId, value) {
+  return {
+    type: CHANGE_VALUE,
+    bunsenId,
+    value
+  }
+}
+
+export function updateValidationResults (validationResult) {
+  // Group errors by input
+  let errors = _.groupBy(validationResult.errors, 'path')
```

Comment:
I dislike the pattern of commenting lines of code like this for two reasons:
1. A line of code should be self-explanatory. If you need a comment to explain what the code does, then Something Is Wrong.
2. Comments are not testable and are often ignored when refactoring, so if a dev refactors line 19, there's a good chance they will ignore line 18 and so the comment will simply be wrong and misleading.

Here are some suggestions for how to deal with this anti-pattern:

1. Can you use different variable names so that these functions are more obvious?

```
const errorsList = validationResult.errors
const inputSelector = 'path'
let errorsByInput = _.groupBy(errorsList, inputSelector)
```

2. Write a one-line function which contains the comment, e.g.:
```
function groupErrorsByInput(errorsList) {
  return _.groupBy(errorsList, 'path')
}
...

let errors = groupErrorsByInput(validationResult.errors)
```